scrapers/VRBangers.yml

```diff
@@ -25,16 +26,18 @@ xPathScrapers:
         selector: //div[contains(@class,"second-text")]/div//text()
         concat: " "
       Tags:
-        Name: //div[starts-with(@class,"video-item__tags ")]//a/text()
+        Name: //a[contains(@href,"category/")]/text()
       Performers:
-        Name: //div[starts-with(@class, 'video-item__info-starring')]//div[starts-with(@class,"tooltip")]//a/text()
+        Name: //div[starts-with(@class, 'video-item__info-starring')]//a/text()
       Studio:
         Name: &studioName
           selector: //meta[@name="dl8-customization-brand-name"]/@content
           postProcess:
             - replace:
                 - regex: \#
                   with: ""
+            - map:
+                VR Bangers: "VRBangers"    
```

Comment:
StashDB is wrong here, I'll submit an edit, this should have a space between the words.

As a side note, I think whether or not a studio has a space between words is often more of the job for a studio alias in Stash than the scraper itself.

Reply:
Edit to fix the StashDB side here https://stashdb.org/edits/21b90e1f-dc7b-4500-8cea-11838411af76

Reply:
I have removed the map then :)
Thanks for your insight.

Reply:
Mapping should be removed as @halorrr  suggests